core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ImageCatalogV4Controller.java

```diff
@@ -188,7 +189,7 @@ public ImageV4Response getSingleImageByCatalogNameAndImageId(Long workspaceId, @
 
     @Override
     @AccountIdNotNeeded
-    @DisableCheckPermissions
+    @InternalOnly
```

Comment:
This endpoint is going to be called from CDP CLI. Will InternalOnly annotation allows CDP CLI to call it?

Reply:
that's why I have added you (or Laszlo) to the PR :) when Laszlo added these annotations,I have helped him and that time, CDP CLI called it with internal actor, we also suggested for @TheTinkerDad to make it internal

Reply:
based on this, it will be internal actor always in case of image catalog: https://github.infra.cloudera.com/thunderhead/thunderhead/blob/2f9cff00aad1ff04ee968b9d160054be53a20f38/services/api/imagecatalog/src/main/java/com/cloudera/thunderhead/service/imagecatalogapi/server/CloudbreakApiClientFactory.java#L39

Reply:
Thanks for the clarification.